modules/ce_etherpad/src/Form/EtherpadSettingsForm.php

```diff
@@ -49,6 +50,11 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#description' => $this->t('Enter Etherpad API Key. You can find API Key in APIKEY.txt on root directory of Etherpad.'),
       '#default_value' => $config->get('etherpad_api_key'),
     ];
+    $form['test_connection'] = array(
+      '#type' => 'submit',
+      '#value' => t('Test connection'),
+      '#submit' => array('::testConnection'),
```

Comment:
Test Connection and Submit button should be in the same row

Reply:
When I make changes to API_URL and click on test connection button, it loads the default value in the API_URL, not the updated value.